Makefile

```diff
@@ -199,6 +202,12 @@ kind-load-test-artifacts: $(KIND) #EXHELP Load the e2e testdata container images
 	$(CONTAINER_RUNTIME) build testdata/bundles/registry-v1/package-with-webhooks.v1.0.0 -t  localhost/testdata/bundles/registry-v1/package-with-webhooks:v1.0.0
 	$(KIND) load docker-image localhost/testdata/bundles/registry-v1/package-with-webhooks:v1.0.0 --name $(KIND_CLUSTER_NAME)
 
+registry-load-bundles: ## Load selected e2e testdata container images created in kind-load-bundles into registry
```

Comment:
Given this - do we still need `kind-load-test-artifacts`? It seems like we can get rid of it.

Reply:
You might be right.